src/ZendRouter.php

```diff
@@ -9,7 +9,7 @@
 
 use Fig\Http\Message\RequestMethodInterface as RequestMethod;
 use Psr\Http\Message\ServerRequestInterface as PsrRequest;
-use Zend\Expressive\Exception;
+use Zend\Expressive\Router\Exception;
```

Comment:
Ah, zend-router is a dependency and it has a [Zend\Expressive\Router\Exception\RuntimeException](https://github.com/webimpress/zend-expressive-router/blob/master/src/Exception/RuntimeException.php).

So your change is actually correct.